libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -559,12 +559,9 @@ bool SRV_Channels::find_channel(SRV_Channel::Aux_servo_function_t function, uint
 /*
   get a pointer to first auxiliary channel for a channel function
 */
-SRV_Channel *SRV_Channels::get_channel_for(SRV_Channel::Aux_servo_function_t function, int8_t default_chan)
+SRV_Channel *SRV_Channels::get_channel_for(SRV_Channel::Aux_servo_function_t function)
 {
     uint8_t chan;
-    if (default_chan >= 0) {
```

Comment:
@IamPete1 can you confirm we don't need this line anymore because add_motor_num() sets up the default for the tail servo?

Reply:
Correct, we were already calling `add_motor_num` which does the defaulting.

https://github.com/ArduPilot/ardupilot/blob/999eb5b03f80ac0ebfe256d834c8349861bd4b1f/libraries/AP_Motors/AP_Motors_Class.cpp#L202-L212